install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -0,0 +1,201 @@
+your cluster's current configuration, or see the advanced installation method on
+how to
+xref:../install_config/install/advanced_install.adoc#adding-nodes-advanced[run
+the playbook for adding new nodes directly].
+////
+
+[IMPORTANT]
+====
+The recommended maximum number of nodes is 300.
+====
+
+To add nodes to your installation:
+
+. Re-run the installer with the `install` subcommand in interactive or
+unattended mode:
++
+----
+$ atomic-openshift-installer [-u] [-c </path/to/file>] install
+----
+
+. The installer detects your current environment and allows you to either
+add an additional node or re-perform a clean install:
++
+====
+----
+Gathering information from hosts...
+Installed environment detected.
+By default the installer only adds new nodes to an installed environment.
+Do you want to (1) only add additional nodes or (2) perform a clean install?:
+----
+====
++
+Choose (1) and follow the on-screen instructions to complete your desired task.
+
+[[adding-nodes-advanced]]
+== Adding Hosts Using the Advanced Install
+
+If you installed using the advanced install, you can add new hosts to your
+cluster by running the *_scaleup.yml_* playbook. This playbook queries the
+master, generates and distributes new certificates for the new hosts, then runs
+the configuration playbooks on the new hosts only. Before running the
+*_scaleup.yml_* playbook, complete all prerequisite
+xref:../install_config/install/prerequisites.html#host-preparation[host
+preparation] steps.
+
+
+ifdef::openshift-enterprise[]
+This process is similar to re-running the installer in the
+xref:adding-nodes-or-reinstalling-quick[quick installation method to add nodes],
+however you have more configuration options available when using the advanced
+method and when running the playbooks directly.
+endif::[]
+
+You must have an existing inventory file (for example, *_/etc/ansible/hosts_*)
+that is representative of your current cluster configuration in order to run the
+*_scaleup.yml_* playbook.
+ifdef::openshift-enterprise[]
+If you previously used the `atomic-openshift-installer` command to run your
+installation, you can check *_~/.config/openshift/.ansible/hosts_* for the last
+inventory file that the installer generated, and use or modify that as needed as
+your inventory file. You must then specify the file location with `-i` when
+calling `ansible-playbook` later.
+endif::[]
+
+[IMPORTANT]
+====
+The recommended maximum number of nodes is 300.
+====
+
+To add a host to an existing cluster:
+
+. Ensure you have the latest playbooks by updating the *atomic-openshift-utils*
+package:
++
+----
+# yum update atomic-openshift-utils
+----
+
+. Edit your *_/etc/ansible/hosts_* file and add `new_<host_type>` to the
+*[OSEv3:children]* section:
++
+For example, to add a new node host, add `new_nodes`:
++
+====
+----
+[OSEv3:children]
+masters
+nodes
+new_nodes
+----
+====
++
+To add a new master hosts, add `new_master`.
```

Comment:
s/master/masters

Hosts added to the `[new_masters]` group should also be added to the `[new_nodes]` group when running the master scale up playbook.

Example
```
[masters]
master[1:2].example.com

[new_masters]
master3.example.com

[nodes]
node[1:3].example.com openshift_node_labels="{'region': 'infra'}"
master[1:2].example.com openshift_schedulable=false

[new_nodes]
master3.example.com openshift_schedulable=false
```